crates/runtime/src/config.rs

```diff
@@ -116,18 +116,29 @@ pub struct ClusterConfig {
     #[arg(
         long = "scheduler-url",
         value_name = "SCHEDULER_URL",
-        default_value = "spiced://localhost:50051",
+        default_value = "http://localhost:50051",
         action
     )]
     pub scheduler_url: Url,
 
+    /// Set the API key configured in the scheduler for secure RPC
+    #[arg(long = "cluster-api-key", value_name = "CLUSTER_API_KEY", action)]
+    pub cluster_api_key: Option<String>,
+
     #[arg(
         long = "allow-insecure-connections",
         value_name = "ALLOW_INSECURE_CONNECTIONS",
         required = false,
         action
     )]
     pub allow_insecure_connections: bool,
+
+    /// The path to the CA cert used to validate the server's identity
```

Comment:
This only exists under the REPL client at the moment, because spiced itself is not a client to itself without cluster mode